### PR TITLE
git cloning - closes #29

### LINKED ISSUE #29: Git Cloning Module
Need a module in Git for cloning

### PR DESCRIPTION
**Pull request by tvaidotas**

### CODE REVIEW

File: topics/git/modules/cloning/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Cloning
```
Comment (bob-crutchley) [inline topics/git/modules/cloning/README.md]:
```suggestion
<!--PROPS
{
    "estTime": 10
}
-->
# Cloning
```
Properties for estimated time

File: topics/git/modules/cloning/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+*projects* or use and existing one. In the example below, a new directory is created. </br>
+![Fork >](https://imgur.com/3b4KMCR.png)
+
+After the cloning has been completed, change directory and run `git branch` to check that the active branch has been 
+checked out. </br>
+![Fork >](https://imgur.com/UEAW1EN.png)
+
+## Tasks
+
+### Task 1
```
Comment (bob-crutchley) [inline topics/git/modules/cloning/README.md]:
```suggestion
### Cloning a Repository
```
Just a task number maybe seems a little ambiguous

File: topics/git/modules/cloning/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Once the cloning is done `git fetch` can be executed without additional arguments to get updates for the remote 
+branches. Similarly a `git pull` without arguments would merge the remote branch into the current branch.
+
+### Cloning a repository
+In order to clone a repository there are a couple of prerequisites:
+* Git installed on the machine
+* URL of the repository
+
+To get the URL of the repository you would need to find a repository you would like to have on your machine. Once you 
+you found the repository click on the *Clone or Download* and copy the URL, make sure you copy it with the *.git*
```
Comment (bob-crutchley) [inline topics/git/modules/cloning/README.md]: `make sure you copy it with the *.git*`
I don't think it actually makes much difference by missing out the `.git`?
At least I haven't noticed a difference

File: topics/git/modules/cloning/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+you found the repository click on the *Clone or Download* and copy the URL, make sure you copy it with the *.git* 
+ending. </br>
+![Fork >](https://imgur.com/hkzKOvt.png)
+![Fork >](https://imgur.com/hOQZaFu.png)
+
+Open a terminal and navigate to the directory where you want to have the project saved. You can create a directory 
+*projects* or use and existing one. In the example below, a new directory is created. </br>
+![Fork >](https://imgur.com/3b4KMCR.png)
+
+After the cloning has been completed, change directory and run `git branch` to check that the active branch has been
```
Comment (bob-crutchley) [inline topics/git/modules/cloning/README.md]:
```suggestion
After the cloning has been completed, change to the project directory and run `git branch` to check that the active branch has been 
```

File: topics/git/modules/cloning/README.md (hunk 3 above)
Comment (tvaidotas) [reply topics/git/modules/cloning/README.md]: I though it did

Comment (bob-crutchley) [reply topics/git/modules/cloning/README.md]: Can clone this repo ok without it:
```text
$ git clone https://github.com/bob-crutchley/notes
Cloning into 'notes'...
remote: Enumerating objects: 58, done.
remote: Counting objects: 100% (58/58), done.
remote: Compressing objects: 100% (40/40), done.
remote: Total 4476 (delta 29), reused 29 (delta 11), pack-reused 4418
Receiving objects: 100% (4476/4476), 2.35 MiB | 3.68 MiB/s, done.
Resolving deltas: 100% (2031/2031), done.
````

**Pull request merged**